superset/assets/javascripts/explore/components/ChartContainer.jsx

```diff
@@ -1,362 +0,0 @@
-import $ from 'jquery';
```

Comment:
no biggy, but if you had used `git mv` instead of `mv` it would have made the PR a bit more aligned.